dark/titles.py

```diff
@@ -482,3 +510,15 @@ def tabSeparatedSummary(self, sortOn=None):
                 '%(subjectTitle)s',
             ]) % titleSummary)
         return '\n'.join(result)
+
+    def toDict(self):
+        """
+        Get information about the titles alignments as a dictionary.
```

Comment:
s/titles alignments/TitlesAlignments

Reply:
I try to use regular English in the description of the methods and put the actual classes in `C{...}` in the `@param` sections.